doc/2/guides/getting-started/1-quick-start/index.md

```diff
@@ -4,4 +4,102 @@ type: page
 title: Quick Start
 description: Scaffold your first application and run it
 order: 100
----
\ No newline at end of file
+---
+
+# Quick Start
+
+Kuzzle is a Node.js application that can be installed on every platforms.
+
+It only requires two services: Elasticsearch and Redis.
+
+In this guide we will use Docker and Docker Compose to run those services.
+
+### Prerequisites
+
+ - [Node.js >= 12](https://nodejs.org/en/download/)
+ - [Docker](https://docs.docker.com/engine/install/)
+ - [Docker Compose](https://docs.docker.com/compose/install/)
+
+#### Kourou
+
+Throughout this guide, we will need to use [Kourou](https://github.com/kuzzleio/kourou), the Kuzzle CLI.
+
+You can install Kourou globally by using NPM: `npm install -g kourou`
+
+#### Admin Console
+
+We are also gonna use the [Admin Console](/core/2/some-page-about-ac) which allows to manage your data, your users and your rights.
+
+::: info
+The Admin Console is a Single Page Application written in Vue.js and using the Javascript SDK.  
+No data related to your connection to Kuzzle will pass through our servers.
+:::
+
+First, we need to setup a new connection to a Kuzzle application. Open the [Admin Console](http://console.kuzzle.io) in your browser and then fill the form as follow:
+
+![Admin Console create connection form](./admin-console-create-connection.png)
+
+Then click on `Create Connection`. 
+
+Your connection is now ready but you can't use it yet since Kuzzle is not running!
```

Comment:
well... the reader hasn't started kuzzle yet, so maybe the admin console part should come later?

Reply:
That's what we call "Mettre la charrue avant les boeufs"